app/auth0/lib/index.ts

```diff
@@ -3,7 +3,7 @@ export * from "./decodeAuthorizationHeader"
 
 // REVISIT: Update with a valid JWT token if you are running Auth0 lib tests
 export const VALID_JWT_TOKEN =
```

Comment:
Generally speaking, this is a TERRIBLE idea 🤣 However...I'm not using testing for CI/CD, so this is a convenience for me.